webpack.config.js

```diff
@@ -25,11 +25,7 @@ module.exports = {
     editorStyle: './assets/src/scss/editorStyle.scss',
     bootstrap: './assets/src/scss/bootstrap-build.scss',
     "country-selector": './assets/src/scss/partials/country-selector.scss',
-    "navigation-bar": './assets/src/scss/partials/navigation-bar.scss',
```

Comment:
Remove all related to the `navigation-bar` since we are not going to use anymore them as `partials`.